test/metrics/prometheus.js

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2018 Jaeger Author.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+import { assert } from 'chai';
+import PrometheusMetricsFactory from '../../src/metrics/prometheus';
```

Comment:
since the factory is exposed via top-level index.js, it is possible to import it from there, as a form of test?